contracts/BSCValidatorSet.sol

```diff
@@ -395,6 +396,9 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
   /*********************** Internal Functions **************************/
 
   function checkValidatorSet(Validator[] memory validatorSet) private pure returns(bool, string memory) {
+    if (validatorSet.length > MAX_NUM_OF_VALIDATORS){
+      return (false, "number of validators exceed limit");
```

Comment:
can we change the comment to:

`the number of validators exceeds the limit`

Reply:
sure